web/src/components/MoreOptionsDropdown.tsx

```diff
@@ -0,0 +1,215 @@
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+import { useFeatureFlag } from 'features/feature-flags/api';
+import { useShare } from 'hooks/useShare';
+import { useAtomValue } from 'jotai';
+import { Link, ShareIcon } from 'lucide-react';
+import { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FaFacebook, FaLinkedin, FaReddit, FaSquareXTwitter } from 'react-icons/fa6';
```

Comment:
We use FaXTwitter in the other social links, would probably be good to use the same one here.

Reply:
Yeah, it's probably good to standardize on one or the other. I'll check with @Alportan

Reply:
The issue with just the XTwitter one is that it looks different than all the others, while XTwitterSquare fits better in this case. 🤔 I would maybe go towards using XTwitterSquare everywhere, as I see that on the buttons we do use the regular ones for Facebook and LinkedIn, which also have some shape in the background 🙌

Reply:
<img width="248" alt="Screenshot 2024-10-24 at 16 51 14" src="https://github.com/user-attachments/assets/0c1c70d7-e3ed-44fe-97c5-ac4dee9f97c3">

Reply:
What if we only use icons here too? 🤔 It would also help on smaller screens

Reply:
Hmm, do you mean something like this?
<img width="260" alt="Screenshot 2024-10-25 at 10 57 05" src="https://github.com/user-attachments/assets/d1365fef-6cca-4110-b2ab-af9b9eddd66e">
<img width="200" alt="Screenshot 2024-10-25 at 10 57 47" src="https://github.com/user-attachments/assets/5753c06f-fea3-41a6-ac60-48c77a5f37fd">
<img width="195" alt="Screenshot 2024-10-25 at 11 01 25" src="https://github.com/user-attachments/assets/eca78a09-0708-4cba-9965-17408748239e">
<img width="253" alt="Screenshot 2024-10-25 at 11 01 02" src="https://github.com/user-attachments/assets/0da3cc81-d60f-4c97-b48c-5bda6e3a72b6">

Reply:
Aaa, I was thinking about the ranking panel ones, but this is interesting 🤔

Reply:
Though I think we're keeping the dropdown list for now - checking with @Alportan to confirm

Reply:
Yes 🙌  Let's keep the interaction fool-proof by keeping the label for context. Accessibility first! 🤓